Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -3,11 +3,12 @@ Copyright (c) 2018 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau, Mario Carneiro, Johan Commelin, Amelia Livingston, Anne Baanen
 -/
-import Mathlib.RingTheory.SimpleRing.Basic
 import Mathlib.Algebra.Field.Equiv
+import Mathlib.Algebra.Field.Subfield.Basic
 import Mathlib.Algebra.Order.Field.Rat
 import Mathlib.Algebra.Order.Ring.Int
 import Mathlib.RingTheory.Localization.Basic
+import Mathlib.RingTheory.SimpleRing.Basic
```

Comment:
I am a little concerned about this new import. Maybe we can create a new file?

Reply:
See the comment here: https://github.com/leanprover-community/mathlib4/pull/18868#issuecomment-2472913388

Reply:
Note that the new import is `Mathlib.Algebra.Field.Subfield.Basic`. The `Mathlib.RingTheory.SimpleRing.Basic` is just reordered alphabetically.

Reply:
Ah ok!